lib/src/util/safe_render_manager/safe_render_manager_helper.dart

```diff
@@ -57,24 +54,6 @@ class SafeRenderManagerHelperComponent extends UiStatefulComponent2<SafeRenderMa
   render() {
     if (state.content == null) return null;
 
-    return cloneElement(state.content, domProps()..ref = chainRef(state.content, _contentRef));
+    return cloneElement(state.content, domProps()..ref = chainRefs(state.content.ref, props.contentRef));
   }
-
-  void _contentRef(ref) {
-    props.contentRef?.call(ref);
-  }
-}
-
-// AF-3369 This will be removed once the transition to Dart 2 is complete.
-// ignore: mixin_of_non_class, undefined_class
-class SafeRenderManagerHelperProps extends _$SafeRenderManagerHelperProps with _$SafeRenderManagerHelperPropsAccessorsMixin {
```

Comment:
I upgraded the boilerplate while I was in here, but this should be fine this component is completely internal